exporter/stats/stackdriver/stackdriver_test.go

```diff
@@ -148,8 +149,8 @@ func TestExporter_makeReq(t *testing.T) {
 }
 
 func TestEqualAggWindowTagKeys(t *testing.T) {
-	key1, _ := tag.NewKey("test-key-one")
-	key2, _ := tag.NewKey("test-key-two")
+	key1, _ := tag.NewKey("test_key_one")
```

Comment:
We also need some validation checks to the NewKey, are any rules here overlapping with the validation check we will implement for NewKey?